### PR TITLE
Add Santa Locator page

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary

Adds a mock "Santa Locator" page to promote Mozilla VPN. It'll be promoted mostly on social media so the page should hopefully look good and on mobile.

This needs to go live before 24 December (obviously).

## Testing

http://localhost:8000/santalocator/
https://www-demo3.allizom.org/santalocator/

### CODE REVIEW

File: bedrock/mozorg/urls.py [hunk 1]
```diff
@@ -142,6 +142,7 @@
         name="mozorg.about.webvision.full",
     ),
     page("analytics-tests/", "mozorg/analytics-tests/ga-index.html"),
+    page("santalocator/", "mozorg/santa-locator.html"),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py:145]: Suggestion (non-blocking): would it be worth adding a redirect for `santa-locator` -> `santalocator` ?

File: media/css/mozorg/santa-locator.scss [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        border: 2px solid $color-white;
+        width: 300px;
+        height: 20px;
+        margin: $spacing-md auto;
+        position: relative;
+        overflow: hidden;
+        color: $color-black;
+
+        &::before {
+            background: repeating-linear-gradient(-45deg, $color-white, $color-white 10px, transparent 10px, transparent 20px);
```
Comment (alexgibson) [inline media/css/mozorg/santa-locator.scss:99]: 🔥

**Review by alexgibson — APPROVED** [review]
Love this - worked well for me on both desktop and mobile r+

**Comment by craigcook** [comment]
Added the redirect and also rebased to fix a merge conflict. Thanks!

**Pull request merged**